salt_lsp/parser.py

```diff
@@ -462,7 +462,9 @@ def __init__(self: Parser, document: str) -> None:
         self._block_starts: List[
             Tuple[
                 Union[
-                    yaml.BlockMappingStartToken, yaml.BlockSequenceStartToken
+                    yaml.BlockMappingStartToken,
+                    yaml.BlockSequenceStartToken,
+                    yaml.FlowSequenceStartToken,
```

Comment:
You surely should also add `FlowMappingStartToken` to the list here since we'll face similar issues with `{}` flow syle blocks